vignettes/ChromBand.Rmd

```diff
@@ -0,0 +1,578 @@
+---
+title: "Using Chromosome Bands as Categories"
+author: 
+   - name: D. Sarkar
+   - name: S. Falcon
+   - name: R. Gentleman
+   - name: Kemmy MO Jones
+     affiliation: [ Chromosome Bands Vignette translation from Sweave to R Markdown / HTML ]
+date: "`r format(Sys.Date(), '%B %d, %Y')`"
+package: Category
+vignette: >
+  %\VignetteIndexEntry{Using Chromosome Bands as Categories}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document:
+    toc: false
+    theme: journal
+bibliography: 
+    - cat.bib
+link-citations: true
+---
+
+# Chromosome bands
+
+Typically, in higher organisms, each chromosome has a centromere and two arms. The short arm is called the p arm and the longer arm the q arm. Chromosome bands (see Figure [1](#fig:chr12ideogram){reference-type="ref" reference="fig:chr12ideogram"}) are identified by differential staining, usually with Giemsa-based stains, and many disease-related defects have been mapped to these bands; such mappings have played an important role in classical cytogenetics. With the availability of complete sequences for several genomes, there have been efforts to link these bands with specific sequence locations [@FureyHaussler]. The estimated location of the bands in the reference genomes can be obtained from the UCSC genome browser, and data linking genes to particular bands can be obtained from a variety of sources such as the NCBI. This vignette demonstrates tools that allow the use of categories derived from chromosome bands, that is, the relevant categories are determined *a priori* by a mapping of genes to chromosome bands.
+
+Figure [1](#fig:chr12ideogram){reference-type="ref" reference="fig:chr12ideogram"} shows an ideogram of human chromosome 12, with the band 12q21 shaded. As shown in the figure, 12q21 can be divided into more granular levels 12q21.1, 12q21.2, and 12q21.3. 12q21.3 can itself be divided at an even finer level of resolution into 12q21.31, 12q21.32, and 12q21.33. Moving towards less granular bands, 12q21 is a part of 12q2 which is again a part of 12q. We take advantage of this nested structure of the bands in our analysis.
+
+```{r,echo=FALSE, eval=FALSE}
+if (!requireNamespace("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+BiocManager::install("Category")
+```
+
+::: Schunk
+::: Sinput
+\> library("Category")\
+\> library("ALL")\
+\> library("hgu95av2.db")\
+\> library("annotate")\
+\> library("genefilter")\
+\> ##library("SNPchip")\
+\> library("karyoploteR")\
+\> library("geneplotter")\
+\> library("limma")\
+\> library("lattice")\
+\> library("graph")\
+:::
+:::
```

Comment:
Convert lines 36-50 into a code block. The code for the figures depends on this code.

It should resemble the following

    ```{r setup}
    library("Category")
    library("ALL")
    ...
    ```